au/code/au/zero.hh

```diff
@@ -17,6 +17,8 @@
 #include <chrono>
 #include <type_traits>
 
+#include "au/fwd.hh"
```

Comment:
Is this needed?

Reply:
I tried to include the forward declaration file in every file whose types are forward declared.  This should give us better test coverage for cases where the forward declaration goes out of sync with the true definition, such as if we forward declare as `class Zero;` but define as `struct Zero {...};`.

That said, I tried creating exactly this error condition, and I wasn't able to get a warning to come up from `//au:zero_test` alone.  I _do_ get errors from some other test in `//...:all`, but I get these regardless of whether this file includes the `fwd.hh` file or not.

All in all: I think it's a good best practice, and I'm going to keep it for that reason, although I'm mystified about why I can't seem to trigger the warnings as easily as I would expect.